### PR TITLE
[BrokenLinksH2] Fixed broken links

### PR DESCRIPTION
**Pull request by kturetz**
Fixed broken links in iis-docs.

### CODE REVIEW

**Comment by kturetz** [comment]
**Global effort to fix broken links** 

@walterov 
@johndeu 
@Rick-Anderson 
@wadepickett  

The Content & Learning team is fixing broken links on docs.microsoft.com for the rest of H2. This effort will eliminate potential accessibility, security, and usability issues. This PR includes only link fixes and does not change other content. I’d very much appreciate your timely review and feedback on the link changes I’ve suggested. Thanks!

File: iis/media/smooth-streaming/microsoft-media-platform-content-map.md [hunk 1]
```diff
@@ -92,4 +92,4 @@ This portal page is designed to help you to deploy the Microsoft Media Platform
...
+| **Get help** | Post questions and ask for troubleshooting guidance from other users and Microsoft team members on the following forums:<br> - [Expression Encoder Forum](https://social.msdn.microsoft.com/forums/expression/en-us/home?forum=encoder)<br> - [IIS Media Forum](https://forums.iis.net/1145.aspx)<br> - [Silverlight.net Forums](https://social.msdn.microsoft.com/Forums/silverlight/en-US/home)<br><br> Get assistance from [Microsoft Support](https://support.microsoft.com/common/international.aspx?RDPATH=dm;en-us;select&amp;target=assistance) for the following products (charges may apply): <br>- Expression Encoder (`https://support.microsoft.com/ph/12897`)<br> - [IIS Media Services](https://www.iis.net/media) 3 and 4 (includes Live Smooth Streaming, Smooth Streaming, Bit Rate Throttling, and Web Playlists)<br> - [Smooth Streaming Client](https://www.iis.net/downloads/microsoft/smooth-streaming) 1.0, 1.1, 1.5, and 2.0<br> - [Advanced Logging](https://www.iis.net/downloads/microsoft/advanced-logging) 1.0<br> - [Microsoft Silverlight](https://www.microsoft.com/Silverlight/) |
```
Comment (wadepickett) [inline iis/media/smooth-streaming/microsoft-media-platform-content-map.md:95]:
```suggestion
| **Get help** | Post questions and ask for troubleshooting guidance from other users and Microsoft team members on the following forums:<br> - [Expression Encoder Forum](https://social.msdn.microsoft.com/forums/expression/home?forum=encoder)<br> - [IIS Media Forum](https://forums.iis.net/1145.aspx)<br> - [Silverlight.net Forums](https://social.msdn.microsoft.com/Forums/silverlight/home)<br><br> Get assistance from [Microsoft Support](https://support.microsoft.com/common/international.aspx?RDPATH=dm;select&amp;target=assistance) for the following products (charges may apply): <br>- Expression Encoder (`https://support.microsoft.com/ph/12897`)<br> - [IIS Media Services](https://www.iis.net/media) 3 and 4 (includes Live Smooth Streaming, Smooth Streaming, Bit Rate Throttling, and Web Playlists)<br> - [Smooth Streaming Client](https://www.iis.net/downloads/microsoft/smooth-streaming) 1.0, 1.1, 1.5, and 2.0<br> - [Advanced Logging](https://www.iis.net/downloads/microsoft/advanced-logging) 1.0<br> - [Microsoft Silverlight](https://www.microsoft.com/Silverlight/) |
```

**Review by wadepickett — APPROVED** [review]
Approved.  Removed "en-US" in a few places so the user's browser has a chance to get a proper local if available.

File: iis/media/windows-media-services/microsoft-media-platform-encoding-and-serving-choices-and-migration-considerations.md [hunk 2]
```diff
@@ -394,7 +394,7 @@ Currently, you can live stream using IIS Media Services 4.1, or use the faciliti
...
 
-Ad Insertion is currently available in client SDKs, with support for the [VMAP](https://en.wikipedia.org/wiki/Vector_Map) and [VPAID](http://www.iab.net/iab_products_and_industry_services/508676/compliance/679310) standards in clients using [MMPPF](http://smf.codeplex.com/), [PlayerFramework](http://playerframework.codeplex.com/), [OSMF AD plugins](http://www.osmf.org/partner.php) and the [Windows Azure Player Framework for iOS](https://github.com/WindowsAzure/azure-media-player-framework).
+Ad Insertion is currently available in client SDKs, with support for the [VMAP](https://en.wikipedia.org/wiki/Vector_Map) and [VPAID](http://www.iab.net/iab_products_and_industry_services/508676/compliance/679310) standards in clients using [MMPPF](https://blogs.windows.com/windowsdeveloper/2013/02/20/microsoft-media-platform-player-framework/), [PlayerFramework](https://blogs.windows.com/windowsdeveloper/2013/02/20/microsoft-media-platform-player-framework/), OSMF AD plugins (`http://www.osmf.org/partner.php`) and the [Windows Azure Player Framework for iOS](https://azure.microsoft.com/en-us/blog/cross-post-announcing-windows-azure-media-player-framework-preview-for-ios/).
```
Comment (wadepickett) [inline iis/media/windows-media-services/microsoft-media-platform-encoding-and-serving-choices-and-migration-considerations.md:397]:
```suggestion
Ad Insertion is currently available in client SDKs, with support for the [VMAP](https://en.wikipedia.org/wiki/Vector_Map) and [VPAID](http://www.iab.net/iab_products_and_industry_services/508676/compliance/679310) standards in clients using [MMPPF](https://blogs.windows.com/windowsdeveloper/2013/02/20/microsoft-media-platform-player-framework/), [PlayerFramework](https://blogs.windows.com/windowsdeveloper/2013/02/20/microsoft-media-platform-player-framework/), OSMF AD plugins (`http://www.osmf.org/partner.php`) and the [Windows Azure Player Framework for iOS](https://azure.microsoft.com/blog/cross-post-announcing-windows-azure-media-player-framework-preview-for-ios/).
```

File: iis/media/windows-media-services/microsoft-media-platform-encoding-and-serving-choices-and-migration-considerations.md [hunk 3]
```diff
@@ -468,19 +468,19 @@ To make direct REST API calls rather than using an SDK, or to use programming la
...
 #### Smooth Streaming Client SDKs
 
-Smooth Streaming Client SDKS are available for a wide variety of platforms, including [Windows 8](https://visualstudiogallery.msdn.microsoft.com/04423d13-3b3e-4741-a01c-1ae29e84fea6?SRC=Home), [Silverlight](https://www.microsoft.com/download/details.aspx?id=29940), [Windows Phone](https://www.iis.net/downloads/microsoft/smooth-streaming), iOS, and Android. Some are available from third parties. For more information, see [Media Services Client Development](https://social.msdn.microsoft.com/Forums/MediaServices/thread/e9092ec6-2dfc-44cb-adce-1dc935309d2a).
+Smooth Streaming Client SDKS are available for a wide variety of platforms, including [Windows 8](https://visualstudiogallery.msdn.microsoft.com/04423d13-3b3e-4741-a01c-1ae29e84fea6?SRC=Home), [Silverlight](https://www.microsoft.com/download/details.aspx?id=29940), [Windows Phone](https://www.iis.net/downloads/microsoft/smooth-streaming), iOS, and Android. Some are available from third parties. For more information, see [Media Services Client Development](https://social.msdn.microsoft.com/Forums/en-US/e9092ec6-2dfc-44cb-adce-1dc935309d2a/media-services-client-development?forum=MediaServices).
```
Comment (wadepickett) [inline iis/media/windows-media-services/microsoft-media-platform-encoding-and-serving-choices-and-migration-considerations.md:471]:
```suggestion
Smooth Streaming Client SDKS are available for a wide variety of platforms, including [Windows 8](https://visualstudiogallery.msdn.microsoft.com/04423d13-3b3e-4741-a01c-1ae29e84fea6?SRC=Home), [Silverlight](https://www.microsoft.com/download/details.aspx?id=29940), [Windows Phone](https://www.iis.net/downloads/microsoft/smooth-streaming), iOS, and Android. Some are available from third parties. For more information, see [Media Services Client Development](https://social.msdn.microsoft.com/Forums/e9092ec6-2dfc-44cb-adce-1dc935309d2a/media-services-client-development?forum=MediaServices).
```

File: iis/media/windows-media-services/microsoft-media-platform-encoding-and-serving-choices-and-migration-considerations.md [hunk 4]
```diff
@@ -508,21 +508,21 @@ The Xbox LIVE Application Development Kit (ADK), available to [Xbox Partners](ht
...
+- [IIS Smooth Streaming Technical Overview](/iis/media/on-demand-smooth-streaming/smooth-streaming-technical-overview)
 
 - [Ingest Assets with the Media Services REST API](https://msdn.microsoft.com/library/windowsazure/jj129593.aspx)
 
 - [Ingesting Assets in Bulk (Windows Azure Media Services SDK)](https://msdn.microsoft.com/library/windowsazure/jj853022.aspx)
 
 - [Ingesting Assets in Bulk with the REST API](https://msdn.microsoft.com/library/windowsazure/jj853021.aspx)
 
-- [Media Services Client Development](https://social.msdn.microsoft.com/Forums/MediaServices/thread/e9092ec6-2dfc-44cb-adce-1dc935309d2a)
+- [Media Services Client Development](https://social.msdn.microsoft.com/Forums/en-US/e9092ec6-2dfc-44cb-adce-1dc935309d2a/media-services-client-development?forum=MediaServices)
```
Comment (wadepickett) [inline iis/media/windows-media-services/microsoft-media-platform-encoding-and-serving-choices-and-migration-considerations.md:519]:
```suggestion
- [Media Services Client Development](https://social.msdn.microsoft.com/Forums/e9092ec6-2dfc-44cb-adce-1dc935309d2a/media-services-client-development?forum=MediaServices)
```

File: iis/media/windows-media-services/microsoft-media-platform-encoding-and-serving-choices-and-migration-considerations.md [hunk 5]
```diff
@@ -550,15 +550,15 @@ The Xbox LIVE Application Development Kit (ADK), available to [Xbox Partners](ht
...
+- [Transform Manager 1.1 download](https://www.iis.net/downloads/microsoft/transform-manager)
 
 - [Unifying Global Video Strategies: MP4 File Fragmentation for Broadcast, Mobile and Web Delivery (white paper)](http://184.168.176.117/reports-public/Adobe/20111116-fMP4-Adobe-Microsoft.pdf)
 
 - [Using the Windows Azure Storage Services](https://msdn.microsoft.com/library/windowsazure/ee924681.aspx)
 
 - [Windows Azure Identity](https://www.windowsazure.com/home/features/identity/)
 
-- [Windows Azure Media Player Framework for iOS](https://github.com/WindowsAzure/azure-media-player-framework)
+- [Windows Azure Media Player Framework for iOS](https://azure.microsoft.com/en-us/blog/cross-post-announcing-windows-azure-media-player-framework-preview-for-ios/)
```
Comment (wadepickett) [inline iis/media/windows-media-services/microsoft-media-platform-encoding-and-serving-choices-and-migration-considerations.md:561]:
```suggestion
- [Windows Azure Media Player Framework for iOS](https://azure.microsoft.com/blog/cross-post-announcing-windows-azure-media-player-framework-preview-for-ios/)
```

File: iis/media/windows-media-services/microsoft-media-platform-encoding-and-serving-choices-and-migration-considerations.md (hunk 4 above)
Comment (kturetz) [reply iis/media/windows-media-services/microsoft-media-platform-encoding-and-serving-choices-and-migration-considerations.md]: Thanks, Wade!

**Pull request merged**